openlibrary/templates/covers/add.html

```diff
@@ -59,7 +59,7 @@
 
             <div class="formElement">
                 <div class="label">
-                    <label id="imageWeb" for="imageUrl">$:_("<span class=\"highlight\"><u>Or</u></span>, paste in the image URL if it's on the web.")</label>
+                    <label id="imageWeb" for="imageUrl">$:_("Or, paste in the image URL if it's on the web.")</label>
```

Comment:
```suggestion
                    <label id="imageWeb" for="imageUrl">$_("Or, paste in the image URL if it's on the web.")</label>
```

Reply:
We don't need the `:` any more because we don't need to render the inner html. However @cdrini mentions we may now have to update our i18n files 

e.g.
https://github.com/internetarchive/openlibrary/blob/3489bc114ca56c8a1286aefce4a4635f78d133cd/openlibrary/i18n/cs/messages.po

Reply:
I will update de Spanish i18n file after this fix.